packages/react-components/src/Modal/styled.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017-2021 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { BodyProps, ContentProps } from './types';
+
+import styled from 'styled-components';
+
+export const Overlay = styled.div`
```

Comment:
Don't like these types of component, as explained above. It really, really makes refactoring an absolute nightmare. As much as styled-component recommends exactly this approach, it is not something that gets used here.

Reply:
✅